src/pkg/packager/compose.go

```diff
@@ -392,31 +365,19 @@ func (p *Packager) getSubPackage(packagePath string) (importedPackage types.Zarf
 }
 
 // Prefix file path with importPath if original file path is not a url.
-func (p *Packager) getComposedFilePath(prefix string, path string) (string, error) {
+func (p *Packager) getComposedFilePath(prefix string, path string) string {
 	message.Debugf("packager.getComposedFilePath(%s, %s)", prefix, path)
 
 	// Return original if it is a remote file.
 	if utils.IsURL(path) {
-		return path, nil
+		return path
 	}
 
-	// Add prefix for local files.
```

Comment:
If the error on create gives back the full path this should be fine

Reply:
Resolving this as this is how it prints for files that don't exist
![image](https://github.com/defenseunicorns/zarf/assets/3977569/3d85553d-fadd-4f1b-88c7-2ef982a448c2)